tasks/bioinfo_utils.wdl

```diff
@@ -544,3 +544,48 @@ task convertCRAMtoFASTQ {
         docker: "quay.io/biocontainers/samtools:1.14--hb421002_0"
     }    
 }
+
+
+task kmerCountingKMC {
+    input {
+        File input_read_file_1
+        File? input_read_file_2
+        String output_file_name
+
+        String working_directory
+        Int kmer_length
+        Int max_ram = 64
+	    Int nb_cores = 16
+        Int disk_size = 200
```

Comment:
maybe make this disk size dynamic based on the input read file. Like in other tasks, with `size(input_read_file_1, "G")`